src/mmw/js/src/modeling/models.js

```diff
@@ -221,6 +221,54 @@ var ScenariosCollection = Backbone.Collection.extend({
             }
             model.set('active', active);
         });
+    },
+
+    duplicateScenario: function(cid) {
+        var source = this.get(cid),
+            sourceMods = source.get('modifications').models,
+            newModel = new ScenarioModel({
+                is_current_conditions: false,
+                name: this.makeNewScenarioName('Copy of ' + source.get('name'))
+            });
+
+        newModel.set('modifications', new ModificationsCollection(sourceMods, {
+            model: ModificationModel
+        }));
```

Comment:
Would removing this code help? https://github.com/caseypt/model-my-watershed/blob/0bb927a9d4ce81f3d7e10ba4fc3768363bee5609/src/mmw/js/src/modeling/models.js#L199